### PR TITLE
DPDK path not found by Lisav2 script fixed

### PR DESCRIPTION
**Pull request by abhimarathe**
Fixed DPDK path not found bug in lisav2 for .git and .tar source input.

### CODE REVIEW

**Comment by juhlee-microsoft** [comment]
Please post the local test result

File: Testscripts/Linux/dpdkSetupAndRunTest.sh [hunk 1]
```diff
@@ -92,11 +92,14 @@ if [ -z "${dpdkSrcLink}" ]; then
 fi
 
 # set DPDK_DIR global
-DPDK_DIR="dpdk"
+ DPDK_DIR="dpdk"
```
Comment (iamshital) [inline Testscripts/Linux/dpdkSetupAndRunTest.sh]: Formatting issue.

**Review by iamshital — CHANGES_REQUESTED** [review]

**Comment by abhimarathe** [comment]
image

**Comment by abhimarathe** [comment]
image

**Comment by abhimarathe** [comment]
image

**Comment by abhimarathe** [comment]
image

File: Testscripts/Linux/dpdkSetupAndRunTest.sh (hunk 1 above)
Comment (abhimarathe) [reply Testscripts/Linux/dpdkSetupAndRunTest.sh]: Done. 
Thank you!

**Comment by LiliDeng** [comment]
@abhimarathe what is the reason for abort or fail?

**Comment by abhimarathe** [comment]
@LiliDeng I'm looking into that. Lisav2 did not generate any error messages and we don't have a 100% repro rate for most of them. One thing I'm sure of is that the abort/fail did not occurre due to this change.

**Comment by juhlee-microsoft** [comment]
> @LiliDeng I'm looking into that. Lisav2 did not generate any error messages and we don't have a 100% repro rate for most of them. One thing I'm sure of is that the abort/fail did not occurre due to this change.

@abhimarathe if abort/fail is not related to this PR, then please investigate through a new issue.

File: Testscripts/Linux/dpdkSetupAndRunTest.sh [hunk 2]
```diff
@@ -94,9 +94,12 @@ fi
 # set DPDK_DIR global
 DPDK_DIR="dpdk"
 if [[ $dpdkSrcLink =~ .tar ]]; then
-	DPDK_DIR="dpdk-$(echo ${dpdkSrcLink} | grep -Po "(\d+\.)+\d+")"
+	#DPDK_DIR="dpdk-$(echo ${dpdkSrcLink} | grep -Po "(\d+\.)+\d+")"
```
Comment (juhlee-microsoft) [inline Testscripts/Linux/dpdkSetupAndRunTest.sh]: Remove the commented line

File: Testscripts/Linux/dpdkSetupAndRunTest.sh [hunk 3]
```diff
@@ -94,9 +94,12 @@ fi
...
+	dpdkSrcTar="${dpdkSrcLink##*/}"
```
Comment (juhlee-microsoft) [inline Testscripts/Linux/dpdkSetupAndRunTest.sh]: 98 and 101 are duplicated

**Review by juhlee-microsoft — CHANGES_REQUESTED** [review]

File: Testscripts/Linux/dpdkSetupAndRunTest.sh (hunk 2 above)
Comment (abhimarathe) [reply Testscripts/Linux/dpdkSetupAndRunTest.sh]: Done

File: Testscripts/Linux/dpdkSetupAndRunTest.sh (hunk 3 above)
Comment (abhimarathe) [reply Testscripts/Linux/dpdkSetupAndRunTest.sh]: Removed duplicated code.

**Review by lizzha — APPROVED** [review]

**Review by LiliDeng — APPROVED** [review]

**Comment by LiliDeng** [comment]
Ava is doing testing.

**Comment by LiliDeng** [comment]
@abhimarathe Could you please run this case VERIFY-DPDK-COMPLIANCE? We found that is pass in master branch, failed in your branch.

**Pull request merged**